pkg/prowgen/prowgen.go

```diff
@@ -100,17 +99,25 @@ func GenerateJobs(configSpec *cioperatorapi.ReleaseBuildConfiguration, info *Pro
 			presubmitTargets = append(presubmitTargets, "[release:latest]")
 		}
 		jobBaseGen := newJobBaseBuilder().TestName("images")
+		if info.Config.MultiArch {
```

Comment:
You could add this logic inside the builder once and for all

Reply:
We don't want that. We need to mutate only the images presubmits and postsubmit.

Reply:
Got it